### ISSUE TITLE
vm is not defined

### ISSUE THREAD

**Issue by jesperlandberg** [pirony/ks-vue-scrollmagic#2]
EDIT: Changed vm to this=)

However now I have another issue. It seems I can't reference this.$gsap.TweenMax.to.... However if i import TweenMax from 'gsap' in the component it works, and just use TweenMax.to....

**Comment by pirony** [comment]
Hi @jesperlandberg ,

It should work: [fiddle](https://jsfiddle.net/romainPouchol/w3zbvtbc/1/)

Which version are you using? (npm or github).
Can you share your code plz? thanks

**Comment by pirony** [comment]
Closing issue due to inactivity. Feel free to open a new one if needed. 
Btw, avoid requiring gsap as it will include it twice in your final bundle... use this.$gsap.TweenMax instead

**Comment by rwfresh** [comment]
First off thanks for making this. I'm having the same problem.  I am using nuxt, I have included ks-vue-scrollmagic, I have created the plugin file in the plugin folder.  There is a typo btw in the project page instructions. src: in nuxt.config.js should be 
```'~/plugins/ksvuefp'```  
and not 
```'~/plugins/ksvuescrollmagic'```  

In my index page this (must use this, not vm for nuxt "cli" created projects) the reference to this.$gsap is empty. Not sure where to go from here. Hope we can re-open this one.
